protos/flyteidl/core/types.proto

```diff
@@ -73,6 +81,9 @@ message LiteralType {
 
         // A blob might have specialized implementation details depending on associated metadata.
         BlobType blob = 5;
+
+        // Defines an enum with pre-defined string values.
+        EnumType enum_type = 7;
```

Comment:
Should it be under Simple?

Reply:
Nope, Simple types are just enums (i.e) they are declared only using an enum value. While enums have values associated with them. The transport is actually just a string